src/store/CategoricalDisplayStore.js

```diff
@@ -272,11 +272,12 @@ export default class CategoricalDisplayStore {
     }
 
     @action.bound setScrollPanel(element) {
-        this.scrollPanel =  element.querySelector('.simplebar-content');
+        this.scrollPanel =  element;
```

Comment:
@MahboobehMohammadi `element` maybe `null` in some case, please handle it also

Reply:
@shayanbinary in which cases it might be null?